test/test_torch.py

```diff
@@ -9745,20 +9745,22 @@ def test_cdist_norm_batch(self, device):
                             self.assertTrue(torch.allclose(expected, actual))
 
     def test_cdist_large(self, device):
-        for cm in ['use_mm_for_euclid_dist_if_necessary', 'use_mm_for_euclid_dist', 'donot_use_mm_for_euclid_dist']:
-            x = torch.randn(1000, 10, device=device)
-            y = torch.randn(1000, 10, device=device)
-            actual = torch.cdist(x, y, p=2, compute_mode=cm)
-            expected = self._brute_cdist(x, y, p=2)
-            self.assertTrue(torch.allclose(expected, actual))
+        if self.device_type in ('cpu', 'cuda'):
```

Comment:
Hi @emcastillo , would you mind removing this `if` check in the PR? 
I'm get these tests skipped in `pytorch/xla` repo (so that we revisit and fix the failures later). 
If we skip `xla` device here, it'll be very hard to discover failures in the future. 
Thanks!

Reply:
Hi! 
The problem here is that in these two tests it says that `cdist` is not compatible with XLA, hence the failure. This is expected as there is an device check in `cdist` that throws an exception if device != cuda or cpu.

Do you have any clue of why these two tests are executed while other cdist tests seem to be ok?

Reply:
I saw the PR that set this tests to be ignored! 
Thanks